### DISCUSSION TITLE
how to specify styles in markdown metadata

### DISCUSSION THREAD

**Discussion by sandys** [mdx-js/mdx#1752] [Q&A] [answered]
hi,
I'm using DaisyUI (https://daisyui.com/) in my website to render MDX pages. 

Now daisyui has a bunch of themes - https://daisyui.com/docs/default-themes/
I want to be able to specify the theme to be used on a per-page (basically per-mdx file) basis. Is there a way to do this ? I realise i might have to do some MDXProvider, but im not able to figure this out. 

Is there a way to add a front-matter section, pick up the styles from there and apply them, etc ?

**Comment by ChristianMurphy** [comment, marked as answer]
MDX includes exports.

You could add the theme as an exported constant

```mdx
export const theme = 'halloween'

# rest of document

goes here
```

You could also use a frontmatter if you prefer https://v2.mdxjs.com/guides/frontmatter/

In both cases, the layout component wrapping MDX would apply the theme.
